src/NoFrixion.MoneyMoov/Models/Payouts/BatchPayout.cs

```diff
@@ -20,6 +20,13 @@ public class BatchPayout
 {
     public Guid ID { get; set; }
 
+    /// <summary>
+    /// This field is used when returning a batch payout record to a client. If set it holds the URL
+    /// the user needs to visit in order to complete a strong authentication check in order to approve 
+    /// the batch payouts.
+    /// </summary>
+    public string? BatchApproveUrl { get; set; }

```

Comment:
Can this be called 'ApproveUrl'? E.g. BatchPayout.ApproveUrl

Reply:
Yes, I'll change that